### DISCUSSION TITLE
Why is gpu+cpu slower than cpu only ?

### DISCUSSION THREAD

**Discussion by franklin050187** [oobabooga/text-generation-webui#1186] [Q&A] [answered]
I did run vicuna 13b model on gpu + cpu with 0.5 tokens / s while CPU only get me 4 tokens / s.
Why is there this difference in speed ?

GPU : 3060TI
CPU : Ryzen 5900X
RAM : 32gb

Also answer from GPU + CPU was way more accurate and coherent than CPU only, but having tried 13b model, I dont see me going back to 7b models... I guess I will wait or buy a 3060 (12gb)

**Comment by BetaDoggo** [comment, marked as answer]
The "auto-devices" option is a bit misleading. The option is actually only offloading the model to system memory. Only the gpu is being used for the actual compute in this mode.
